### DISCUSSION TITLE
Language change

### DISCUSSION THREAD

**Discussion by euotiniel** [angolaosc/.github#3] [Ideas]
First of all to congratulate you for the great initiative that has been taken, recently I walked that not crazy looking for open source projects, and here in Angola almost nothing is found (as far as I could get).

But precisely because it is an Angolan community, perhaps we could rethink the language we are using. English is an international language, yes. But maybe we should really change the language to **Portuguese** because of "globalization here." 

However, if English is kept, we will still be here to contribute, thanks!

**Comment by antonio-pedro99** [comment]
Hello @euotiniel sounds interesting. I am moving this to discussion so that we can talk about it further.

**Reply by euotiniel** [reply]
Good!

**Comment by antonio-pedro99** [comment]
I am gonna get back to you soon.

**Comment by antonio-pedro99** [comment]
a

**Reply by David-the-trader** [reply]
hey guys, I am David, an Angolan living in Namibia, for the past 8 years, nice to meet you.
